superdesk-common/package.json

```diff
@@ -10,13 +10,16 @@
     "/dist/src"
   ],
   "scripts": {
+    "watch": "tsc --watch",
     "build": "tsc && ts-node bundle-css.ts",
     "watch-css": "ts-node bundle-css.ts --watch",
     "test": "mocha",
     "debug-tests": "mocha --inspect-brk",
     "prepublish": "rm -rf dist && npm run build && npm run test"
   },
   "devDependencies": {
+    "@types/classnames": "^2.3.1",
+    "classnames": "^2.3.2",
```

Comment:
doesn't look used. If it was though - it'd be required during runtime - thus it should be in main dependencies - not dev dependencies